component builder missing a dependency

## Description
Using the component builder, we ran into an issue where we were missing @spectrum-css/vars
Should probably be a dependency of some sort?


## Steps to reproduce

1. Clone react spectrum
2. remove package.json reference to spectrum-css/vars and maybe from yarnlock as well
3. yarn install
4. build storybook

```
(node:1146) UnhandledPromiseRejectionWarning: Error: Cannot find module '@spectrum-css/vars'
Require stack:
- /home/circleci/react-spectrum-v3/node_modules/@spectrum-css/component-builder/css/lib/varUtils.js
- /home/circleci/react-spectrum-v3/node_modules/@spectrum-css/component-builder/css/plugins/postcss-custom-properties-mapping.js
- /home/circleci/react-spectrum-v3/node_modules/@spectrum-css/component-builder/css/processors.js
- /home/circleci/react-spectrum-v3/postcss.config.js
- /home/circleci/react-spectrum-v3/.storybook-v3/webpack.config.js
- /home/circleci/react-spectrum-v3/node_modules/@storybook/core/dist/server/utils/server-require.js
- /home/circleci/react-spectrum-v3/node_modules/@storybook/core/dist/server/common/custom-presets.js
- /home/circleci/react-spectrum-v3/node_modules/@storybook/core/dist/server/config.js
- /home/circleci/react-spectrum-v3/node_modules/@storybook/core/dist/server/build-static.js
- /home/circleci/react-spectrum-v3/node_modules/@storybook/core/server.js
- /home/circleci/react-spectrum-v3/node_modules/@storybook/react/dist/server/build.js
- /home/circleci/react-spectrum-v3/node_modules/@storybook/react/bin/build.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:772:15)
    at Function.resolve (internal/modules/cjs/helpers.js:74:19)
    at Object.<anonymous> (/home/circleci/react-spectrum-v3/node_modules/@spectrum-css/component-builder/css/lib/varUtils.js:110:47)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
    at Function.Module._load (internal/modules/cjs/loader.js:703:12)
    at Module.require (internal/modules/cjs/loader.js:830:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at Object.<anonymous> (/home/circleci/react-spectrum-v3/node_modules/@spectrum-css/component-builder/css/plugins/postcss-custom-properties-mapping.js:14:16)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
    at Function.Module._load (internal/modules/cjs/loader.js:703:12)
    at Module.require (internal/modules/cjs/loader.js:830:19)
    at require (internal/modules/cjs/helpers.js:68:18)
(node:1146) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:1146) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

## Expected behavior
<!-- Describe what you expected to happen -->


## Screenshots
<!-- If applicable, add screenshots to help explain the problem -->


## Environment
 - **Spectrum CSS version:** <!-- 2.13.0 -->
 - **Browser(s) and OS(s):** <!-- Chrome 75.0.3770.142 on Win 10 -->

## Additional context
<!-- Provide any additional information that might help us debug the issue -->

--- a/tools/component-builder/package.json
+++ b/tools/component-builder/package.json
@@ -11,6 +11,9 @@
     "url": "https://github.com/adobe/spectrum-css/issues"
   },
   "license": "Apache-2.0",
+  "peerDependencies": {
+    "@spectrum-css/vars": "^2.0.2"
+  },
   "dependencies": {
     "autoprefixer": "^6.5.3",
     "browser-sync": "^2.26.7",

